pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+		xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>BBFW</groupId>
+	<artifactId>traffic_simulation</artifactId>
+	<version>1.0-SNAPSHOT</version>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<kotlin.version>1.1.2</kotlin.version>
+		<main.class>org.codefx.demo.bingen.energy_model.MainKt</main.class>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.jetbrains.kotlin</groupId>
+			<artifactId>kotlin-stdlib</artifactId>
+			<version>${kotlin.version}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<sourceDirectory>${project.basedir}/src/main/kotlin</sourceDirectory>
+		<testSourceDirectory>${project.basedir}/src/test/kotlin</testSourceDirectory>
```

Comment:
Git kann nur Dateien, leere Ordner funktionieren da nicht.

Am Ende jedes Pull Requests sollte das Projekt in einem konsistenten Zustand sein. Hier Tests zu deklarieren, die es noch nicht gibt und Dependencies zu deklarieren, die nicht benötigt werden (JUnit, weiter oben) erfüllt diese Bedingung nicht. :smiley: Nehmt bitte alle testbezogenen Teile aus der POM raus.